internal/jsoncanonicalizer/es6numfmt.go

```diff
@@ -0,0 +1,71 @@
+//
+//  Copyright 2006-2019 WebPKI.org (http://webpki.org).
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+// This package converts numbers in IEEE-754 double precision into the
+// format specified for JSON in EcmaScript Version 6 and forward.
+// The core application for this is canonicalization:
+// https://tools.ietf.org/html/draft-rundgren-json-canonicalization-scheme-02
+
+package jsoncanonicalizer
```

Comment:
I'm a little hesitant about copy/pasting all of this code into the go-witness codebase. I'm also curious if we even need to canonicalize the manifest at all

Reply:
So we generate a digest of all the manifests captured in the step. I suppose the question is, do we need it? My answer here is, "we _could_ have it, but it doesn't really mean anything". Are we going to use when querying / verifying? Probably not.